docs/docs-content/user-management/authentication/api-key/delete-api-key.md

```diff
@@ -92,3 +96,248 @@ learn more about deleting an API key as a tenant admin.
 
 </TabItem>
 </Tabs>
+
+## API
+
+You can use the Palette API to delete an API key programmatically. Use the
+`https://api.spectrocloud.com/v1/apiKeys/:uid`[endpoint](https://docs.spectrocloud.com/api/v1/v-1-api-keys-uid-delete)
+to delete an API key.
+
+To delete an API key, you first must have the API key's unique identifier. Use the following steps to learn how to
+delete an API key programmatically.
```

Comment:
```suggestion
You can use the Palette API with the `https://api.spectrocloud.com/v1/apiKeys/:uid` [endpoint](https://docs.spectrocloud.com/api/v1/v-1-api-keys-uid-delete) and the API key's unique identifier to delete an API key programmatically. 

Use the following steps to learn how to delete an API key.
```
maybe? :) to avoid repetition